app/controllers/reservations_controller.rb

```diff
@@ -394,6 +394,20 @@ def upcoming_today?(reservation)
     (reservation.reserve_start_at.to_date == now.to_date || reservation.reserve_start_at < now) && reservation.reserve_end_at > now
   end
 
+  # Some browsers are not updating their cached JS and have an out-of-date calendar
+  # JS library which uses unix timestamps for the parameters. This allows us to handle both.
+  def parse_time_param(value)
+    return unless value
+
+    if value.to_s.match?(/\A\d{10}\z/) # unix timestamp
+      Time.zone.at(value.to_i)
+    elsif value.to_s.match?(/\A\d{13}\z/) # unix timestamp with milliseconds
+      Time.zone.at(value.to_i / 1000)
```

Comment:
I believe this branch is unnecessary, because both `start` and `end` come in as UNIX timestamps with second precision (1524373200 and 1524978000). The only param that comes in with millisecond precision is `_` (1524755233886), which we don’t attempt to process in any way.